### ISSUE TITLE
Keys containing slashes (`/`) are not set correctly.

### ISSUE THREAD

**Issue by elussich** [anaptfox/cacheman-file#9]
I was trying to store cache values using URLs as keys; my best guess is that `/` characters were producing an error while trying to save the file, because they were interpreted as part of the path expression.

Here's an error sample:
```
Error: ENOENT: no such file or directory, open '/Users/username/project/tmp/cacheman:/api/path/sample?param=123.json' 
```
where `/api/path/sample?param=123` was the key used.

A possible solution/suggestion would be to sanitize (for example, through `querystring` NodeJS native module, or [sanitize-filename](https://www.npmjs.com/package/sanitize-filename) npm module) any passed keys, before convert them into file names.

Let me know and I can put up a quick PR.
Thanks in advance!

**Comment by anaptfox** [comment]
Yes!  A PR would be awesome!

**Comment by elussich** [comment]
Pull request #10 created.
Let me know your thoughts!

**Comment by theworkflow** [comment]
Closed via #10
